pkgs/by-name/xr/xrescat/package.nix

```
@@ -0,0 +1,49 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  gnumake,
+  gcc,
+  gdb,
+  xorg,
+  SDL2,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "xrescat";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "regolith-linux";
+    repo = "xrescat";
+    rev = "e8e261441682244112b2020e2ad102768e6ba3fd";
+    sha256 = "0mMcoNNkaFO6O0F8HjIA8Q8MtfSHLeXY9cGkVd83Vls=";
+  };
+
+  buildInputs = [
+    gnumake
+    gcc
+    gdb
+    xorg.libX11.dev
```

```suggestion
    libX11
```
You can import `libX11` directly (`.dev` is also not necessary)